features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2264,6 +2273,30 @@
             <Permissions>/permission/admin/manage/identity/idvp/view</Permissions>
             <Scopes>internal_idvp_view</Scopes>
         </Resource>
+        <Resource context="(.*)/api/server/v1/api-resources/.search" secured="true" http-method="POST">
+            <Permissions>/permission/admin/manage/identity/apiresourcemgt/view</Permissions>
+            <Scopes>internal_api_resource_view</Scopes>
```

Comment:
are these apiresourcemgt permissions are registered in the permission tree already? Also is that permissions to scope mappings are defined

Reply:
Defined https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2159 here